tests/src/kakarot/test_state.py

```diff
@@ -40,6 +45,35 @@ def test_existing_account(
         def test_not_in_state(self, cairo_run):
             cairo_run("test__is_account_alive__not_in_state")
 
+    class TestIsAccountWarm:
+        def test_account_in_state(self, cairo_run):
```

Comment:
naming should be `test_should_return_true_when_account_in_state` or something shorter if you can find it, but the test name should desribe the test